lib/activerecord-import/import.rb

```diff
@@ -225,13 +225,31 @@ def support_setting_primary_key_of_imported_objects?
     #
     # ====  Using A Hash
     #
-    # The :on_duplicate_key_update option can be a hash of column name
+    # The :on_duplicate_key_update option can be a hash of column names
     # to model attribute name mappings. This gives you finer grained
     # control over what fields are updated with what attributes on your
     # model. Below is an example:
     #
     #   BlogPost.import columns, attributes, :on_duplicate_key_update=>{ :title => :title }
     #
+    # == On Duplicate Key Update (Postgres 9.5+)
+    #
+    # The :on_duplicate_key_update option is a hash with :key and
+    # :columns attributes. Below is an example:
+    #
+    #   BlogPost.import columns, values, :on_duplicate_key_update=>{ :key => :id, :columns => [ :date_modified, :content, :author ] }
```

Comment:
I wonder if we should provide additional example(s) for `:key` usage. The ones that come to mind are:

* Default: no `key` specified. (If this ends up being applicable)
* Passing `key` to use existing index
* Passing `key` to use existing column(s) as index? (I think this is what you have above)

There may be other options such as passing in PG-compatible `index_expression`(s) but I'm not entirely sure if that will already work or if it'd be more work (which I don't think should block this PR getting in).